### ISSUE TITLE
Error: connect ECONNREFUSED 127.0.0.1:3000

### ISSUE THREAD

**Issue by zeke** [probot/smee#38]
Here's my URL: https://smee.io/AZ0KUUSusyualwme

Events are trickling into the browser just fine, but events cause the CLI to spit out this error:

```
$ smee -u "https://smee.io/AZ0KUUSusyualwme"
Forwarding https://smee.io/AZ0KUUSusyualwme to http://127.0.0.1:3000/
Connected https://smee.io/AZ0KUUSusyualwme


{ Error: connect ECONNREFUSED 127.0.0.1:3000
    at Object._errnoException (util.js:1041:11)
    at _exceptionWithHostPort (util.js:1064:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1153:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 3000,
  response: undefined }
```

```
$ cat /usr/local/lib/node_modules/smee-client/package.json | json version
1.0.1
```

**Comment by manusrivastav** [comment]
START server
cannot call  http://localhost:3000 0 5 { Error: connect ECONNREFUSED 127.0.0.1:3000
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 3000 }

**Comment by hashtagchris** [comment]
@manusrivastav Do you have any processes listening on port 3000? If you're using a different port you can use the `-p` option to customize the port.

**Comment by runleonarun** [comment]
@bkeepers now that smee is listening for events, I get this same error that @zeke reported. Note, I see them fine in the browser too. 

```
 smee --url https://smee.io/JllG9WDZKJbkVnsI --path /event_handler --port 3000
Forwarding https://smee.io/JllG9WDZKJbkVnsI to http://127.0.0.1:3000/event_handler
Connected https://smee.io/JllG9WDZKJbkVnsI
{ Error: connect ECONNREFUSED 127.0.0.1:3000
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1163:14)
  errno: 'ECONNREFUSED',
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 3000,
  response: undefined }
{ Error: connect ECONNREFUSED 127.0.0.1:3000
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1163:14)
  errno: 'ECONNREFUSED',
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 3000,
  response: undefined }
```

**Comment by runleonarun** [comment]
Note, this worked when I added the following line to the server.rb:
`set :bind, '0.0.0.0'`

**Comment by dumbitdownjr** [comment]
Did anyone find a fix for this issue? `set: bind, '0.0.0.0` was already set for me in server.rb

**Comment by MartinTrenkovIBM** [comment]
One possible solution is. For example if you have a flask application and if you only run the service to start smee, but you HAVENT run your flask script in the terminal, this will occur. start both smee AND the script
